readme.md

```diff
@@ -28,6 +29,12 @@ $ rake test # run the test suite
 $ rake test:all # run the test suite on all supported Rubies
 ```
 
+### SDK CLI Shell Setup
+```bash
+$ cd looker-sdk/shell
+$ bundle install
+```
+
```

Comment:
I'd be inclined to leave this out. There is no context here yet for the shell stuff. We should eventually write more about that. I think a readme in the shell dir would be better.